### PR TITLE
Proofreading Javier's excellent updates - in some places, shortening some things

### PR DESCRIPTION
**Pull request by weaverryan**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.3+ |
| Fixed tickets | #5117 |

Hi guys!

After @javiereguiluz's excellent updates in #5117, I proofread the first section (the most important one) even more. The big takeaway is that I tried to shorten some things - less talking, more showing (the original version was quite "long", Javier shortened some things, I'm proposing going even further).

Thanks!

### CODE REVIEW

File: cookbook/controller/error_pages.rst [hunk 1]
```diff
@@ -71,8 +62,29 @@ templates is based on the HTTP status code and the request format:
...
+    ├─ Resources/
+    │  └─ TwigBundle/
+    │     └─ views/
+    │        └─ Exception/
+    │           ├─ error404.html.twig
+    │           ├─ error403.html.twig
+    │           ├─ error.html.twig      # All other HTML errors (including 500)
+    │           ├─ error404.json.twig
+    │           ├─ error403.json.twig
+    │           ├─ error.json.twig      # All other JSON errors (including 500)
```
Comment (WouterJ) [inline cookbook/controller/error_pages.rst]: I don't it's usefull to show the line from `app/`, we don't do it for the sub dirs

File: cookbook/controller/error_pages.rst [hunk 2]
```diff
@@ -132,14 +145,12 @@ is undefined. The solution is to add the following check before using this funct
 Testing Error Pages during Development
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-One of the biggest hurdles of testing how do custom error pages look in your
-application is the fact that Symfony ignores them in the development environment
-and displays the default exception pages instead.
+Symfony shows the big *exception* page instead of your customized error page
+when you're in the development environem
```
Comment (xabbuh) [inline cookbook/controller/error_pages.rst]: environment.

Comment (xabbuh) [inline cookbook/controller/error_pages.rst]: Actually, with the paragraph below, we can just remove this one completely I guess.

**Comment by javiereguiluz** [comment]
:+1: great work Ryan! This article is vey important, so the shorter and more actionable, the better :)

File: cookbook/controller/error_pages.rst (hunk 2 above)
Comment (weaverryan) [inline cookbook/controller/error_pages.rst]: yep - good catch, that was my intention!

**Pull request merged**